web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -158,6 +159,100 @@ public function search(): array {
+      'advanced-search-preview-headline-text' => $this->t('CQL search string', [], ['context' => 'advanced search']),
+      'advanced-search-preview-empty-text' => $this->t('-', [], ['context' => 'advanced search']),
+      'advanced-search-reset-text' => $this->t('reset', [], ['context' => 'advanced search']),
+      'advanced-search-copy-string-text' => $this->t('copy string', [], ['context' => 'advanced search']),
+      'advanced-search-edit-cql-text' => $this->t('edit cql', [], ['context' => 'advanced search']),
+      'advanced-search-search-button-text' => $this->t('search', [], ['context' => 'advanced search']),
+      'loading-results-text' => $this->t('Loading results...', [], ['context' => 'advanced search']),
+      'to-advanced-search-button-text' => $this->t('Back to advanced search', [], ['context' => 'advanced search']),
+      'cql-search-title-text' => $this->t('CQL search', [], ['context' => 'advanced search']),
+      'copied-to-clipboard-text' => $this->t('Copied', [], ['context' => 'advanced search']),
+      'copied-link-to-this-search-text' => $this->t('Link copied to clipboard', [], ['context' => 'advanced search']),
+      'clause-and-text' => $this->t('and', [], ['context' => 'advanced search']),
+      'clause-or-text' => $this->t('or', [], ['context' => 'advanced search']),
+      'clause-not-text' => $this->t('not', [], ['context' => 'advanced search']),
+      'advanced-search-filter-material-type-text' => $this->t('material type', [], ['context' => 'advanced search']),
+      'advanced-search-filter-literature-form-text' => $this->t('literature form', [], ['context' => 'advanced search']),
+      'advanced-search-filter-access-text' => $this->t('accessibility', [], ['context' => 'advanced search']),
+      'advanced-search-filter-book-text' => $this->t('book', [], ['context' => 'advanced search']),
+      'advanced-search-filter-ebook-text' => $this->t('ebook', [], ['context' => 'advanced search']),
+      'advanced-search-filter-audio-book-text' => $this->t('audio book', [], ['context' => 'advanced search']),
+      'advanced-search-filter-article-text' => $this->t('article', [], ['context' => 'advanced search']),
+      'advanced-search-filter-movie-text' => $this->t('movie', [], ['context' => 'advanced search']),
+      'advanced-search-filter-music-text' => $this->t('music', [], ['context' => 'advanced search']),
+      'advanced-search-filter-physical-text' => $this->t('physical', [], ['context' => 'advanced search']),
+      'advanced-search-filter-online-text' => $this->t('online', [], ['context' => 'advanced search']),
+      'advanced-search-filter-fiction-text' => $this->t('fiction', [], ['context' => 'advanced search']),
+      'advanced-search-filter-non-fiction-text' => $this->t('Non-fiction', [], ['context' => 'advanced search']),
+      'advanced-search-filter-all-text' => $this->t('All', [], ['context' => 'advanced search']),
```

Comment:
Wow. A lot of strings 😅 Great work.
 I recommend sorting the text props alphabetically to help the maintenance of them in the future.